RaceControl/RaceControl/ViewModels/VideoDialogViewModel.cs

```diff
@@ -450,6 +450,11 @@ private void WindowStateChangedExecute(Window window)
             SetWindowed();
             SetFullScreen();
         }
+        // Needed to set the resizemode to 'NoResize' when exiting fullscreen using a Windows key combination
```

Comment:
Comment should be 'CanResize' instead of 'NoResize' I think.

Reply:
Yep, fixed it